docs/modules/clients/pages/go.adoc

```diff
@@ -1,6 +1,8 @@
 = Go Client
 :page-api-reference: https://pkg.go.dev/github.com/hazelcast/hazelcast-go-client@v{page-latest-supported-go-client}
 
+TIP: For the latest Go API documentation, see https://pkg.go.dev/github.com/hazelcast/hazelcast-go-client@v{page-latest-supported-go-client}[Hazelcast Go Client docs].
```

Comment:
Omitting the version links to the latest released version.
```suggestion
TIP: For the latest Go API documentation, see https://pkg.go.dev/github.com/hazelcast/hazelcast-go-client[Hazelcast Go Client docs].
```

Reply:
Ooh, nice tip - I might make that a separate PR and back/forward port it as I just cherry picked these changes from 5.4

Reply:
Though actually we want this version to point to the one that matches the docs, so the latest supported version should come from the antora file and match that branch e.g. 5.4